_data/topnav.yml

```diff
@@ -10,3 +10,6 @@ links:
     page: discuss.md
   - title: wiki
     url: https://fabricmc.net/wiki/
+  - title:
```

Comment:
Would github be better in the footer somewhere? or do we even need it at all?

Whats your thinking behind putting this in the nav bar?

Adding the [docs](https://docs.fabricmc.net/) site here instead might be better use of space.

Reply:
Removed and replaced with the docs link. I added it as I thought it would be helpful but I agree, the header is likely not the right place. I'll do another PR for the footer at some point soon